stdlib/ysh/args.ysh

```diff
@@ -195,13 +206,32 @@ func parseArgs(spec, argv) {
             if (_status !== 0) {
               error "Expected Float after '$arg', got '$[argv[i]]'" (code=2)
             }
+          } elif (flag.type is List[Float]) {
+            setvar i += 1
+            if (i >= len(argv)) {
+              error "Expected Float after '$arg'" (code=2)
+            }
+
+            setvar value = args[flag.name] if (flag.name in args) else []
```

Comment:
Oh how about `get(args, flag.name, [])` everywhere?   I think that is equivalent and shorter

It is like Python's `args.get(flag.name, [])`, except that we can't use a method, so we use the free function get()

Reply:
Oh, yeah! `get` is _much_ better. I'll make that change :+1: